storage/aptosdb/src/lib.rs

```diff
@@ -858,6 +858,7 @@ impl DbReader for AptosDB {
         version: Version,
     ) -> Result<HashMap<StateKey, StateValue>> {
         gauged_api("get_state_values_by_key_prefix", || {
+            error_if_version_is_pruned(&self.ledger_pruner, "State", version)?;
```

Comment:
Are these state kvs so we use ledger pruner instead of state pruner?

Reply:
yes